src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -1097,6 +1097,8 @@ class SeparatorNodePropertiesDescription:
             constants.FLUID_OIL: Scalar("volume fraction", 0.5, "-"),
         }
     )
+    gas_separation_efficiency = attrib_scalar(default=Scalar(1.0, "-"))
+    liquid_separation_efficiency = attrib_scalar(default=Scalar(1.0, "-"))
```

Comment:
I particularly think it's better to put the category explicitly and not rely on the default value (if I'm not mistaken is dimensionless)

Reply:
Done